### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by shuvashish76** [FossifyOrg/General-Discussion#3] [General]
https://hosted.weblate.org/projects/simple-mobile-tools/
@comradekingu  Sorry for ping you here, is there any easier way to transfer ownership or they've to setup everything fresh? _(see https://github.com/SimpleMobileTools/General-Discussion/issues/241)_

**Comment by comradekingu** [comment]
There is, but I am a libre software guy and not paid to help companies I don't like.
Weblate itself has paid support at https://weblate.org/hosting/
With the way it seems the new company is doing things, at least that means some funds for the Weblate coffers.

**Comment by shuvashish76** [comment]
Yes I totally agree with you.
I think you misunderstood the situation. "FossifyX" isn't related to "ZipoApps" the company which bought SMT. It's a community fork by @naveensingh (former SMT member/dev) to keep the original principle of SMT alive.  (see https://github.com/SimpleMobileTools/General-Discussion/issues/241#issuecomment-1837464578)

**Comment by naveensingh** [comment]
It would be great if it could be transferred. It'll save some time and effort.

**Comment by naveensingh** [comment]
Hello @comradekingu, could you please take a moment to review [this comment](https://github.com/orgs/FossifyX/discussions/3#discussioncomment-7744877) made by @shuvashish76 and provide us with an answer?
Thank you.

**Comment by comradekingu** [comment]
@naveensingh

Any Weblate admin can add a new admin in
https://hosted.weblate.org/access/simple-mobile-tools/#users
then change the project name in
https://hosted.weblate.org/settings/simple-mobile-tools/

It is also possible to download everything as a ZIP, and/or copy over all the settings to a new project.

I am [kingu](https://hosted.weblate.org/user/kingu/#managed) on Hosted Weblate, and not actually a simple mobile tools admin (only [tibbi](https://hosted.weblate.org/user/tibbi/) is).

**Reply by naveensingh** [reply]
Got it, thanks :)

**Comment by naveensingh** [comment]
I'm setting up translations here (almost done): https://hosted.weblate.org/projects/fossify

The weblate team is currently busy so they won't review the project for libre hosting approval right away but they have extended the trial period for two months.

**Comment by Mr-Bajs** [comment]
@naveensingh I suggest when ever what name you choose that you lock the translations of the app names, they shouldn't be translated. It led to confusion with the SMT suite. Though launcher names can and should be translated.

**Reply by Aga-C** [reply]
@naveensingh Also, should changelogs be translated too?

**Reply by Mr-Bajs** [reply]
> @naveensingh Also, should changelogs be translated too?

Wouldn't it be best if the previous Simple Mobile Tools changelog where archived, and acssesible that way and Fossify starting from scratch and a new versioning base?

**Reply by naveensingh** [reply]
> @naveensingh I suggest when ever what name you choose that you lock the translations of the app names, they shouldn't be translated. It led to confusion with the SMT suite. Though launcher names can and should be translated.

Yes, app names won't be translated. I'll be moving names to donottranslate soon.

**Reply by naveensingh** [reply]
> @naveensingh Also, should changelogs be translated too?

Currently, no because I'll be deleting the SMT changelogs. The future changelogs _should be_ translated but it's not a necessity.

**Reply by naveensingh** [reply]
> > @naveensingh Also, should changelogs be translated too?
> 
> Wouldn't it be best if the previous Simple Mobile Tools changelog where archived, and acssesible that way and Fossify starting from scratch and a new versioning base?

Old changelogs will be deleted and yes, we are starting from 1.0.0.

**Comment by naveensingh** [comment]
Weblate integration seems to work fine, closing this...
